scripts/kind-build-test.sh

```diff
@@ -190,5 +193,10 @@ echo "==========================================================================
 
 export KUBECONFIG
 
+if [[ "$ENABLE_PROMETHEUS" == true ]]; then
+    echo -n "Setting up prometheus into the cluster ... "
+    setup_prometheus "$CLUSTER_NAME"
+fi
```

Comment:
I think you can remove `scripts/lib/prometheus.sh` entirely and just add this into the `kind-build-test.sh` script above around line 130 where we are loading the other images into the KinD cluster:

```bash
if [[ "$ENABLE_PROMETHEUS" == true ]]; then
    echo -n "Loading prometheus image into the cluster ... "
    kind load docker-image --quiet --name "${CLUSTER_NAME}" \
        --nodes="${CLUSTER_NAME}"-worker,"${CLUSTER_NAME}"-control-plane \
        prom/prometheus || exit 1
    echo "ok."
fi
```

and then here, just do:

```bash
if [[ "$ENABLE_PROMETHEUS" == true ]]; then
    PROMETHEUS_SETUP_DIR=$ROOT_DIR/kind-setup/prometheus
    PROMETHEUS_SETUP_FILE_PATH=$PROMETHEUS_SETUP_DIR/prometheus-setup.yaml
    echo -n "Deploying prometheus into the cluster ... "
    kubectl apply -f "$PROMETHEUS_SETUP_FILE_PATH" 1>/dev/null
    echo "ok."
fi
```
```

Reply:
Updated the code per the comments.